src/_data/sidenav.yml

```diff
@@ -142,8 +142,27 @@
           permalink: /docs/development/tools/android-studio
         - title: Visual Studio Code
           permalink: /docs/development/tools/vs-code
-        - title: Dart DevTools
-          permalink: https://flutter.github.io/devtools/
+        - title: DevTools
+          permalink: /docs/development/tools/devtools
+          children:
+            - title: Overview
+              permalink: /docs/development/tools/devtools/overview
+            - title: From Android Studio
```

Comment:
nit: should we have a subheading of "installing" that contains these three "from" items?

Reply:
The reason I didn’t do that was because we can’t go another level with our side nav. We are limited to this number of levels.